wagtail/images/models.py

```diff
@@ -478,15 +474,11 @@ def find_existing_rendition(self, filter: "Filter") -> "AbstractRendition":
             # (avoiding further cache/db lookups)
             raise Rendition.DoesNotExist
 
-        # Next, query the cache (if configured)
-        try:
-            cache = caches["renditions"]
-            key = Rendition.construct_cache_key(self.id, cache_key, filter.spec)
-            cached_rendition = cache.get(key)
-            if cached_rendition:
-                return cached_rendition
-        except InvalidCacheBackendError:
-            pass
+        # Next, query the cache
+        cache_key = Rendition.construct_cache_key(self, cache_key, filter.spec)
+        cached_rendition = Rendition._get_cache().get(cache_key)
+        if cached_rendition:
+            return cached_rendition
```

Comment:
for example this becomes https://github.com/wagtail/wagtail/pull/9550/files#diff-c195bc8bd4d5a11ae02d0451edb4dd4a8e7b0014398a384a1be3ff3c355a5ee8R477 in #9550